app/controllers/frontend/application_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
```

Comment:
No need to split for frontend, keep all frontend controller in controllers, example: controllers/users_controllers.rb, controllers/products_controllers.rb...